{"ID": "c796a1b8-8cd0-48f5-9150-fba63910f246", "effective_cve": "CVE-2020-26137", "image": {"exact": "docker.io/sonatype/nexus3@sha256:e8fea6b4279f2b5b24b36170459cb7aa3d6afe999f9d3e3713541be28bae8ec4"}, "label": "FP", "note": "The package actually present is the RPM python3-urllib3@1.24.2-4.el8, which is where this vulnerability should be reported.", "package": {"name": "urllib3", "version": "1.24.2"}, "timestamp": "2023-08-16T14:11:23.330913", "tool": "grype@v0.65.1", "user": "willmurphy", "vulnerability_id": "GHSA-wqvq-5m8c-6g24"}